html/changelog.html

```diff
@@ -5,16 +5,16 @@
   <link rel="stylesheet" type="text/css" href="changelog.css">
   <base target="_blank" />
   <script type='text/javascript'>
-  

```

Comment:
# DO NOT ADJUST THE GENERATED CHANGELOG IN ANY WAY

i repeat

# DO NOT ADJUST THE GENERATED CHANGELOG IN ANY WAY